client/sites-dashboard/components/sites-grid-item.tsx

```diff
@@ -62,7 +62,16 @@ export const SitesGridItem = memo( ( { site }: SitesGridItemProps ) => {
 		<SitesGridTile
 			leading={
 				<ThumbnailLink { ...siteDashboardUrlProps }>
-					<SiteItemThumbnail className={ siteThumbnail } site={ site } size={ 'medium' } />
+					<SiteItemThumbnail
+						className={ siteThumbnail }
+						site={ site }
+						size={ 'medium' }
+						mshotsOption={ {
+							vpw: 1200,
+							vph: 1200 * ( 11 / 16 ),
```

Comment:
We could copy the aspect ratio from the CSS property itself (extract it into a variable and use it in both places).